### PR TITLE
Add find tag feature to FindCommand

### PR DESCRIPTION
**Pull request by Dominic-Khoo**
Add functionality for users to filter patients based on medical conditions (labeled as tags) using FindCommand.

Users have to provide the appropriate prefix, e.g find t/diabetic to search for patients with the "diabetic" tag.

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/FindCommand.java [hunk 1]
```diff
@@ -2,27 +2,29 @@
...
 public class FindCommand extends Command {
 
     public static final String COMMAND_WORD = "find";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all persons whose names contain any of "
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all persons whose names or tags contain any of "
             + "the specified keywords (case-insensitive) and displays them as a list with index numbers.\n"
             + "Parameters: KEYWORD [MORE_KEYWORDS]...\n"
-            + "Example: " + COMMAND_WORD + " alice bob charlie";
+            + "Example: " + COMMAND_WORD + " alice bob charlie (for names) OR find t/diabetic t/G6PD-Def";
```
Comment (itsnotsherm) [inline src/main/java/seedu/address/logic/commands/FindCommand.java:23]: there might be a better way to display this usage message but small issue

File: src/main/java/seedu/address/logic/commands/FindCommand.java [hunk 2]
```diff
@@ -2,27 +2,29 @@
 
 import static java.util.Objects.requireNonNull;
 
+import java.util.function.Predicate;
+
 import seedu.address.commons.util.ToStringBuilder;
 import seedu.address.logic.Messages;
 import seedu.address.model.Model;
-import seedu.address.model.person.NameContainsKeywordsPredicate;
```
Comment (itsnotsherm) [inline src/main/java/seedu/address/logic/commands/FindCommand.java:8]: side note I think predicates can have their own directory for organisation

**Review by itsnotsherm — APPROVED** [review]
LGTM, not much to comment on, neat implementation

File: src/main/java/seedu/address/logic/commands/FindCommand.java (hunk 2 above)
Comment (Dominic-Khoo) [reply src/main/java/seedu/address/logic/commands/FindCommand.java:8]: noted, will do in next iteration

**Pull request merged**